### PR TITLE
Update AdminLTE to version 2.2.

### PR DESCRIPTION
**Pull request by tim96**
Update AdminLTE to version 2.2

### CODE REVIEW

**Comment by Soullivaneuh** [comment]
I see you change nothing except vendor files.

* Is that you want? Just upgrade and see later for implement feature?
* Is that working without any needed template modification?

**Comment by tim96** [comment]
> Is that working without any needed template modification?

yes, because we don't change file names.

> Is that you want? Just upgrade and see later for implement feature?

I want to add right block menu, but now we don't have idea which information show there (I have some plans how to use this block in some project). So I deleted right block, because show empty block it is not a good decision.
The next iteration we can add, for example:
* Minified left sidebar with icons.
* Add right block menu with some important information and we can show it to user.
* Add some widgets on dashboard

@Soullivaneuh if you have idea about right block menu, I can implement it.

File: Resources/public/vendor/admin-lte/..bower.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
+{
+  "name": "AdminLTE",
```
Comment (Soullivaneuh) [inline Resources/public/vendor/admin-lte/..bower.json]: I think this file should not be here. `..bower.json` ? Oo

**Comment by Soullivaneuh** [comment]
I think extra feature should be proposed on another PR, if you want to do it.

Just correct my review and we will merge it.

**Comment by tim96** [comment]
@Soullivaneuh I correct my commit based on your review.

**Comment by Soullivaneuh** [comment]
Thank you @tim96.

**Comment by tim96** [comment]
@Soullivaneuh hello. I found a small problem, when tested it on another project. If label for list in side menu is too long, it didn't transfer on new line, because AdminLTE add:
```css
.sidebar-menu,
.main-sidebar .user-panel,
.sidebar-menu > li.header {
  white-space: nowrap!important;
  overflow: hidden;
}
```
old

now

Maybe we can override this features on style.css ?

**Comment by Soullivaneuh** [comment]
@tim96 I can't see your screenshots.

> Maybe we can override this features on style.css ?

If it a bugfix yes. The best is to do a PR on adminLTE repository and put the hack on `style.css` with the PR link.

In this case, we can remove this hack easily when vendor PR will be accepted.

**Comment by tim96** [comment]
> @tim96 I can't see your screenshots.
Update links

> If it a bugfix yes. 
I am not sure that it is a bug.

**Comment by Soullivaneuh** [comment]
Just saw your screenshots.

Indeed, I'm not sure about a bug neither.

I think you should ask for it here: https://github.com/almasaeed2010/AdminLTE

Link our issue inside to get updated. ;-)

**Comment by tim96** [comment]
@Soullivaneuh I create a new issue there.

But now we can something add to override this behavior or not?

**Comment by Soullivaneuh** [comment]
> But now we can something add to override this behavior or not?

If it's not a bug and the expected behavior, we should let it as is IMO.

**Comment by Soullivaneuh** [comment]
Note: You should provide your screenshots on the issue.

**Pull request merged**